package.json

```
@@ -52,6 +52,6 @@
   },
   "support": true,
   "engines": {
-    "node": "^16 || ^14 || ^12"
+    "node": "^22.0.0"
```

To be able to run on the latest LTS
```suggestion
    "node": "^16 || ^14 || ^12 || >=18"
```

---

Yeah this version is better since it isn't breaking.

---

This hasn't had any action in ages, so yeah, if it works for you - you can keep this puppy 😁 

Officially it's owned by the package maintenance working group, but I'm sure membership can be sorted out. It looks like I own https://github.com/wiby if it was decided to move this out.

---

Sure, although well, @wesleytodd or an administrative member will have to give me write access. Currently, I am a ['member'](https://github.com/nodejs/package-maintenance/pull/617) but I don't have the necessary permissions.

---

I am pretty unopinionated about where it lives. @dominykas I assume you are not working on it anymore? It seems easiest to keep it here and add @bjohansebas as a comitter on the repo (an easy change I can make) but we had some policies in place for this and I am not sure how we want to apply them since the project is partly abandoned at the moment. IMO it would be best to just add @bjohansebas, but maybe we would need to run that by the broader group?

---

I understand that you have policies, just like other organizations do, but couldn't this discussion be taken offline? Maybe in an issue? In the end, several of these packages are somewhat abandoned, and I have offered to maintain them because they are useful to me. I have been patient, waiting for a response on whether I can maintain them. If not, I will choose another path.